test/configFile/options.yaml

```diff
@@ -0,0 +1,2 @@
+inverse: true
```

Comment:
We could amend the tests to look for and load a `config.{json|yaml}` in preference to `options.yaml` so the test options don't have to be specified twice?

Reply:
Added